Packs/CommonScripts/Scripts/StixCreator/StixCreator.py

```diff
@@ -175,12 +269,14 @@ def main():
                               raw_response=str(bundle)))
 
     elif len(indicators) == 1:
```

Comment:
Do we have to separate to 2 cases?

Reply:
We need it in order not to break backward compatibility